PF/Assignment 1/assign_1.java

```diff
@@ -0,0 +1,178 @@
+package shopping;
+
```

Comment:
Please use proper naming convention for naming files. You can use ShoppinCart.java instead of assign_1.java.